src/astarte-client/client.ts

```diff
@@ -336,26 +335,21 @@ class AstarteClient {
         groupName: encodedGroupName,
       }),
     );
-
     if (details) {
-      endpointUri.search = new URLSearchParams({ details: true } as any).toString();
+      endpointUri.search = new URLSearchParams({ details: 'true' }).toString();
     }
-
     const response = await this.$get(endpointUri.toString());
     return response.data.map((device: AstarteDeviceDTO) => AstarteDevice.fromObject(device));
   }
 
-  async removeDeviceFromGroup(params: any): Promise<void> {
+  async removeDeviceFromGroup(params: { groupName: string; deviceId: string }): Promise<void> {
     const { groupName, deviceId } = params;
-
     if (!groupName) {
       throw Error('Invalid group name');
     }
-
```

Comment:
I'm not too fond of removing new lines between different `if` statements. The resulting code will look too compact.

Reply:
The method is quite tiny to risk getting it cluttered. I personally prefer having it without blank lines since it's easier to distinguish each method's block when scrolling the file. However I am not strict with such rule, perfectly fine if you prefer blank lines.

Reply:
Then let's avoid editing lines far off the edited lines of the PR

Reply:
That's fine, I placed the blank lines back.